builder/frameworks/arduino.py

```
@@ -24,263 +24,1019 @@
+        "esp32c6": 31600,    # ESP32-C6
+        "esp32h2": 32000,    # ESP32-H2
+        "esp32p4": 32000,    # ESP32-P4
+    }
+
+    default_value = platform_defaults.get(mcu, 31600)
+
+    # Debug output only in verbose mode
+    if logging.getLogger().isEnabledFor(logging.DEBUG):
+        logging.debug(
+             f"Max. possible platform default threshold for {mcu}: "
+            f"{default_value}")
```

Flake8 (7.2.0)

[error] continuation line unaligned for hanging indent

(E131)